marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -362,6 +362,11 @@ public Path getCustomMappingFunctionsDir() {
             IOUtils.closeQuietly(is);
         }
 
+        //New 5.3.0 roles
+        writeRoleFile(rolesDir, "data-hub-load-data-reader.json");
+        writeRoleFile(rolesDir, "data-hub-load-data-writer.json");
+
```

Comment:
This would create roles but it seems incomplete as "data-hub-operator/developer" are not inheriting these roles. The load data artifact is not written with appropriate permissions.